core/src/main/java/gyro/core/validation/CollectionMaxValidator.java

```diff
@@ -0,0 +1,22 @@
+package gyro.core.validation;
+
+import java.util.Collection;
+
+import gyro.core.resource.Diffable;
+
+public class CollectionMaxValidator implements Validator<CollectionMax> {
```

Comment:
Let's use the full form instead of abbreviation, so rename this `CollectionMaximumValidator`.

Reply:
Never mind. we already have `Min` and `Max` annotations.